app/handlers/uploadVideo.py

```diff
@@ -18,5 +26,42 @@ class UploadVideoHandler(tornado.web.RequestHandler):
 
     @apiError error_message The error message to display.
     """
+    def initialize(self):
+        self.file_name  = str(uuid4().int)
+        self.file_path = os.path.realpath(os.path.join('..','.temp',self.file_name))
+        print 'Upload Video Initialized @{}'.format(self.file_path)
+
     def post(self):
-        self.finish("Upload Video")
+        if 'multipart/form-data; boundary=' in self.request.headers['Content-Type']:
+            boundary = self.request.headers['Content-Type'][30:]
+        else:
+            self.finish('Incorrect File Format: Must be of type multipart/form-data')
+
+        files = {}
+        arguments = {}
+        with open(self.file_path, 'rb') as f:
+            try:
+                content = f.read()
+                parse_multipart_form_data(boundary,content,arguments,files)
+            except MemoryError as err:
+                raise tornado.web.HTTPError(reason = 'Memory Error: File too large',\
+                        status_code = 507)
+        if 'identifier' in arguments:
+            identifier = arguments['identifier']
+        video = files['video'][0]
+        vdict = {'video':(video['filename'],video['body'])}
+        project_identifier = api.saveFiles(vdict)
```

Comment:
Maybe add the function you need to this file and call it here, rather than reference api.py?